test/acceptance/display_matching_invocations_alongside_expectations_test.rb

```diff
@@ -0,0 +1,38 @@
+require File.expand_path('../acceptance_test_helper', __FILE__)
+require 'mocha/setup'
+
+class DisplayMatchingInvocationsAlongsideExpectations < Mocha::TestCase
+  include AcceptanceTest
+
+  def setup
+    setup_acceptance_test
+    @original_env = ENV.to_hash
```

Comment:
> What did you have in mind?

Something like:

```ruby
def setup
  @original_options = ENV['MOCHA_OPTIONS']
  ENV['MOCHA_OPTIONS'] = 'verbose'
end

def teardown
  ENV['MOCHA_OPTIONS'] = @original_options
end
```

> I thought that'd be slightly more complicated and involve at least one conditional because ENV may not contain any MOCHA_OPTIONS at all originally.

If `ENV` doesn't contain any `MOCHA_OPTIONS` then `ENV['MOCHA_OPTIONS']` will return `nil`, right? So I don't see why you'd need a conditional. But maybe I'm missing something...?

Reply:
I see - I hadn't considered that. In my mind the distinction isn't likely to be significant, but I'm happy to go with the way you've done it.